documentation/project/contributing/running.md

```diff
@@ -3,11 +3,11 @@
 ### Running locally via docker
 If you have built a container locally or wish to run a public one you could bring everything up like this:
 ```shell
-docker-compose -f docker/kafka-ui.yaml up -d
+docker-compose -f documentation/compose/kafka-ui.yaml up -d
 ```
 
 ### Running locally without docker
-Once you built the app, run the following:
+Once you built the app, run the following: (TODO: is there a way to hot-reload while developing? Does it always require a re-build then a run?)
 
 ```sh
 ./mvnw spring-boot:run -Pprod
```

Comment:
Is there a way to pass a configuration to this command?

I tried this:
```bash
./mvnw clean spring-boot:run -Pprod -Dexec.args="src/main/resources/application-local.yml"
```

Reply:
Thanks for the review @Haarolean!

I tried this, to no luck. Any ideas?
```
 ./mvnw clean spring-boot:run -Pprod  -Dexec.args="--spring.config.location=file:///path/to/conf.yaml"  
```

Reply:
Any ideas @Haarolean or anyone else? Would be nice to have this in the docs.

Reply:
Sorry for the delay.
This should work:
` ./mvnw clean spring-boot:run -Pprod  -Dspring.config.location=file:///path/to/conf.yaml`

Reply:
Thanks! Updated